javascript/exorelay/src/websocket-connector/websocket-connector.ls

```diff
@@ -56,20 +62,27 @@ class WebSocketConnector extends EventEmitter
 
 
   connect: ~>
+    @should-reconnect-on-socket-closed = yes
     @socket = new WebSocket "ws://#{@exocom-host}:#{@exocom-port}/services"
       ..on 'open', @_on-socket-open
       ..on 'message', @_on-socket-message
       ..on 'error', @_on-socket-error
+      ..on 'close', @_on-socket-close
+
+
+  _on-socket-close: ~>
+    @emit 'offline'
+    if @should-reconnect-on-socket-closed
+      @connect!
 
 
   _on-socket-open: ~>
     @emit 'online'
 
 
   _on-socket-error: (error) ~>
-    | error.errno is 'EADDRINUSE'   =>  @emit 'error', "port #{@exocom-port} is already in use"
-    | error.errno is 'ECONNREFUSED' =>  wait 1_000, @connect
```

Comment:
This line was unnecessary for the current tests to pass. It appears the websocket library we are using keeps retrying on the initial connections even though it emits an error.